### PR TITLE
add lime-librenet6 package

### PR DESCRIPTION
**Pull request by aparcar**
Signed-off-by: Paul Spooren <mail@aparcar.org>

### CODE REVIEW

File: packages/lime-librenet6/files/etc/tinc/librenet6/tinc-up [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+ifconfig $INTERFACE up txqueuelen 1000
```
Comment (G10h4ck) [inline packages/lime-librenet6/files/etc/tinc/librenet6/tinc-up]: ifconfig is deprecated! Isn't ip installed by default on openwrt at least some minimal version from busybox ?

File: packages/lime-librenet6/files/etc/uci-defaults/tinc-setup [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    commit tinc
+EOF
+}
+
+# disable ra
+uci set network.lan.accept_ra=0
+uci commit network
+
+# set IPv6 subnet of librenet6.librenet.org
+uci set lime.network.main_ipv6_address '2002:8aff:5905:%N1%N2::/64'
```
Comment (G10h4ck) [inline packages/lime-librenet6/files/etc/uci-defaults/tinc-setup]: This seems to interfere with normal lime behavior or am I missing something?

File: packages/lime-librenet6/files/etc/uci-defaults/tinc-setup [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+# be sure to enable babeld
+uci get lime.network.protocols | grep "babeld" || {
+    uci add_list lime.network.protocols="babeld:17"
+}
+
+# commit modified lime config
+uci commit lime
+
+# apply all changes
+lime-config
```
Comment (G10h4ck) [inline packages/lime-librenet6/files/etc/uci-defaults/tinc-setup]: This two line seems could interfere with normal lime behavior too

**Comment by dangowrt** [comment]
On Tue, Aug 28, 2018 at 08:47:26AM -0700, G10h4ck wrote:
> G10h4ck requested changes on this pull request.
> 
> 
> 
> > @@ -0,0 +1,2 @@
> +#!/bin/sh
> +ifconfig $INTERFACE up txqueuelen 1000
> 
> ifconfig is deprecated! Isn't ip installed by default on openwrt at least some minimal version from busybox ?

Just tried
ip link set dev eth0 txqueuelen 1000
and realized that busybox' ip applet indeed doesn't support setting
txqueuelen. We should change that...

File: packages/lime-librenet6/files/etc/uci-defaults/tinc-setup (hunk 2 above)
Comment (aparcar) [reply packages/lime-librenet6/files/etc/uci-defaults/tinc-setup]: Well you have to use an IP of that subnet if you want a working connection. To keep it simple for users I decided to add this

**Comment by G10h4ck** [comment]
same comments apply to https://github.com/libremesh/lime-packages/pull/400 I wonder if creating a librenet6 lime-proto instead of an uci-default could integrate better with libremesh, being a lime-proto it could also be configurable from /etc/config/lime lime-app etc.

**Comment by dangowrt** [comment]
> +ifconfig $INTERFACE up txqueuelen 1000

please use
```
ip link set $INTERFACE qlen 1000
```
instead (qlen is a synonym for txqueuelen).

**Comment by ilario** [comment]
@aparcar @G10h4ck once the comments are addressed, can we merge this or there is something else?

**Comment by gmarcos87** [comment]
Hi @ilario, I don't think this is suitable for merge. If I run this on our libremesh network it won't work. You should not change the ipv6 settings from lime but add new ones. I think the requests for changes made by gio are quite correct. It would be interesting to reformulate this as lime-proto-librenet6 instead of a script that replaces other configurations already used.
Anyway the package was already added in #400. Maybe we should close this pull-request,

**Comment by ilario** [comment]
I'm confused, does the issues raised by @G10h4ck apply also on #400?
If so, should we open an issue for tracking the need of changing the currently merged code?

**Comment by gmarcos87** [comment]
That's right, we have to open the issue.

**Pull request closed without merging**